interactions/api/models/channel.py

```diff
@@ -1122,6 +1329,10 @@ async def get_history(self, limit: int = 100) -> Optional[List["Message"]]:
         :rtype: List[Message]
         """
 
+        warn(
```

Comment:
Why does this need to be deprecated? The method has nothing wrong with using it, and removing it doesn't simplify code elsewhere

Reply:
It allows less control than the new method (for example if you want to find a specific message but don't know where it is you'll have to get all set messages) and imo it's just better to use the new method

I also compared them, the new one's were around 5 seconds faster than the deprecated

Reply:
If your argument is that the new feature supersedes the old, deprecating it at the very least makes sense. If there's literally no reason to use this over the new methods, why keep it?

Reply:
Because I didn't want to have this breaking